src/main/kotlin/game/commands/ArrowVolleyCommand.kt

```diff
@@ -0,0 +1,19 @@
+package game.commands
+
+import game.interfaces.BaseWarrior
+
+
+class ArrowVolleyCommand(receivers: MutableList<Receiver>) : Command(receivers){
+    override fun execute() {
```

Comment:
why this fun is empty?

Reply:
Fun was abstract in class Command. Fixed it